src/components/LiquidityDepth/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useMemo, useState } from 'react'
+import { Currency, Price, Token } from '@uniswap/sdk-core'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { DarkBlueCard } from 'components/Card'
+import DensityChart from './DensityChart'
+import { RowBetween } from 'components/Row'
+import { Trans } from '@lingui/macro'
+import { AutoColumn } from 'components/Column'
+import { ChartContext } from './hooks'
+import ZoomButtons from './ZoomButtons'
+
+const MIN_ZOOM = 10
+const MAX_ZOOM = 200
```

Comment:
mm these are a little confusing in the context of full range, also if you can't exceed these, the # of ticks we need data for is dramatically smaller